app/Http/Utilities/Correspondence.php

```diff
@@ -14,72 +14,80 @@ class Correspondence
             'type' => 'welcome',
             'key' => '1',
             'label' => 'Welcome',
-            'subject' => 'Welcome, [First Name]!',
-            'body' => "Woohoo, so glad you signed up for my [Campaign Name] competition, [First Name]! Feel free to get a head start on the competition and begin today. \n\rI will be in touch with you with more info in the next few days. Excited to watch you climb the leaderboard and crush the competition! \n\r[Name of Staff]",
+            'subject' => 'Welcome, :first_name!',
+            'body' => "Woohoo, so glad you signed up for my :campaign_name competition, :first_name! Feel free to get a head start on the competition by working towards the greatest number of :reportback_noun :reportback_verb as you can. \n\rI will be in touch with you with more info in the next few days. Excited to watch you climb the leaderboard and crush the competition! \n\r:staff_name",
```

Comment:
not sure if @sbsmith86 is still using the Database names but the `:staff_name` is actually stored as `:sender_name`
my mistake for not changing that in the issue